README.md

```diff
@@ -29,7 +29,7 @@ Currently, Zettle provides APIs for the following markets:
 -   [Product Library](product-library.adoc)
 -   [Inventory](inventory.adoc)
 -   [Image](image.adoc)
--   [Pusher (Webhooks)](pusher.adoc)
+-   [Pusher](pusher-api/api-reference.md)
```

Comment:
should this not point to just the folder: pusher-api/ under which the developers can see all the docs?

Reply:
I updated the pointer to the folder as you suggested. My intention was to point readers a starting point, as they might not know which doc to start. But that was merely an assumption. Later on when we have made an information architecture, let's revisit this topic.